books/free-programming-books-ar.md

```diff
@@ -38,6 +39,11 @@
 * [سبيلك المختصر لتعلم لغة c# - برمجة الواجهات](https://www.mobarmijoun.com/2014/04/c_19.html) - Khaled Al-Saadani (Arab Coders Academy)
 
 
+### Go
+
+* [تعلم لغة Go بسهولة](https://itwadi.com/learn-go-easily) - Alan Donovan, Brian Kernigan, مجموعة عمل (PDF)
```

Comment:
Please do not add this. The link is to a page in Arabic that purports to be an Arabic translation of our book (The Go Programming Language, gopl.io), but everything about it looks wrong: the title and cover art are different, the blurb (back cover text) is unrelated. The link to the PDF appears to be broken, so I cannot tell how similar the actual text is to our work.

This work claims to be a translation of our book, yet also to be licensed under Creative Commons, but those two things cannot both be true: if this is a translation, it is a derivative work, and therefore the copyright (the right to issue licenses) belongs to the authors, not the translator, and we have not issued any Creative Commons license for the text of our book.

I would love to see a good authorized Arabic translation of our book, but unfortunately this is not it.

Reply:
thank you @adonovan  we won't add it.

Reply:
Thanks for feedback! It was strange to me that is not mapped in the translation page, reason why I ask you about :angel: .

Reply:
@adonovan ok this will be removed from the PR. 
the link does work though (https://sourceforge.net/projects/omlx/files/open%20books/1.0/learn-go-easily-arabic-itwadi.com.pdf/download) for the pdf if you want to take a look at the book and want to check it. i can help translate parts to see the similarities and differences between the original and the translated ( i will also do this on my own)

sorry for this.